notary-registration/src/main/kotlin/registration/NotaryRegistrationStrategy.kt

```diff
@@ -13,7 +13,8 @@ import sidechain.iroha.util.ModelUtil
  */
 @Component
 class NotaryRegistrationStrategy(
-    @Autowired private val irohaConsumer: IrohaConsumer
+    @Autowired private val irohaConsumer: IrohaConsumer,
+    @Autowired private val notaryAccount: String
```

Comment:
I'd rather add `@Qualifier`. `String` is a very popular class. Somebody may create another `String` bean by mistake.